sample-code/java/src/IOSCreateSessionTest.java

```diff
@@ -0,0 +1,43 @@
+import io.appium.java_client.ios.IOSDriver;
+import io.appium.java_client.ios.IOSElement;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testng.Assert;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public class IOSCreateSessionTest extends BaseTest {
+    private IOSDriver<WebElement> driver;
+
+    @BeforeSuite
+    public void setUp() throws Exception {
+        File classpathRoot = new File(System.getProperty("user.dir"));
+        File appDir = new File(classpathRoot, "../apps");
+        File app = new File(appDir.getCanonicalPath(), "TestApp.app.zip");
+
+        String deviceName = System.getenv("IOS_DEVICE_NAME");
+        String platformVersion = System.getenv("IOS_PLATFORM_VERSION");
+        DesiredCapabilities capabilities = new DesiredCapabilities();
+        capabilities.setCapability("deviceName", deviceName == null ? "iPhone 6s" : deviceName);
+        capabilities.setCapability("platformVerison", platformVersion == null ? "11.1" : platformVersion);
+        capabilities.setCapability("app", app.getAbsolutePath());
+        capabilities.setCapability("automationName", "XCUITest");
+        driver = new IOSDriver<WebElement>(getServiceUrl(), capabilities);
+    }
+
+    @AfterSuite
+    public void tearDown() {
+        driver.quit();
+    }
+    
+    @Test
+    public void testCreateSession () {
+        // Check that the XCUIElementTypeApplication was what we expect it to be
+        IOSElement applicationElement = (IOSElement) driver.findElementByClassName("XCUIElementTypeApplication");
```

Comment:
I'm not sure one can match the root XCUIElementTypeApplication element by using class name search with xcuitest. Only descendant elements are going to be matched